.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
+          '
+          +build/deprecated,
+          +build/endif_comment,
```

Comment:
Does this mean to add a comment after each `#endif`? If yes, I don't think this should be enforced.

Reply:
Yes, I think that was Google's intent. I'll disable it.